src/getters/get-title.ts

```diff
@@ -0,0 +1,29 @@
+import * as Unist from 'unist';
+
+const { select } = require('unist-util-select');
+const toString = require('mdast-util-to-string');
+
+import { Frontmatter } from '../frontmatter';
+
+// Priority: yaml title, then first heading
+export const getTitle = (data?: Unist.Node) => {
```

Comment:
This function looks fairly similar to `get-summary`. 
No pressure, but we could extract the shared logic to a different fn.